### PR TITLE
Added CPU Usage details to statsd

### PR DESCRIPTION
**Pull request by bah2830**
Added CPU user, system stats as well as the load average stat.

### CODE REVIEW

**Comment by bah2830** [comment]
Is there anything else I need to do to get k8s-bot testing approved?

**Comment by timstclair** [comment]
ok to test

File: storage/statsd/statsd.go [hunk 1]
```diff
@@ -31,6 +31,12 @@ type statsdStorage struct {
 
 const (
 	colCpuCumulativeUsage string = "cpu_cumulative_usage"
+	// CPU system
+	colCpuUsageSystem string = "cpu_usage_system"
+	// CPU user
+	colCpuUsageUser string = "cpu_usage_user"
+	// CPU average load
+	colLoadAverage string = "load_average"
```
Comment (timstclair) [inline storage/statsd/statsd.go]: Would it be better to call this `"cpu_load_average"` to group it?

**Comment by timstclair** [comment]
LGTM, with one question.

File: storage/statsd/statsd.go (hunk 1 above)
Comment (bah2830) [inline storage/statsd/statsd.go]: I don't have a problem with that. I can get it updated when I get home from work.

Thanks

**Pull request merged**